examples/abitest/module_1/rust/src/lib.rs

```diff
@@ -33,19 +33,20 @@ pub extern "C" fn oak_main(handle: u64) -> i32 {
     }
 }
 pub fn main(in_handle: u64) -> i32 {
+    let in_channel = oak::ReadHandle {
+        handle: oak::Handle::from_raw(in_handle),
+    };
```

Comment:
nit: this should move down by 2 lines (initializing logging and setting the panic hook should be the first things done)